patches/server/0372-implement-optional-per-player-mob-spawns.patch

```diff
@@ -693,11 +693,15 @@ index 95e5660e6cb1afb5ebdb3dbbe59a07c879bddb4b..88145f04989c71a686aae1b486087ecd
 +                    minDiff = Math.min(limit - world.getChunkSource().chunkMap.getMobCountNear(entityplayer, enumcreaturetype), minDiff);
 +                }
 +                difference = (minDiff == Integer.MAX_VALUE) ? 0 : minDiff;
++            } else {
++                if (!info.canSpawnForCategory(enumcreaturetype, limit)) {
++                    continue;
++                }
```

Comment:
I think I can remove `if (!info.canSpawnForCategory(enumcreaturetype, limit))`,
because `info.canSpawnForCategory(enumcreaturetype, limit)` is equal to `difference > 0` when per-player-mob-spawns = false.